I'm not sure why this would be, but the test fails for me if I specifically do `pytest tests/test_consistency.py` but passes if I do just `pytest` see below. 

```
(base) 15:16:15 ericfoot pyDeltaRCM_WMT $ (amoodie/test_consistent) pytest tests/test_consistent.py 
================================== test session starts ===================================
platform darwin -- Python 3.7.7, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
rootdir: /Users/ericfoot/Dropbox/programs/git_repos/pyDeltaRCM_WMT
plugins: arraydiff-0.3, remotedata-0.3.2, hypothesis-5.11.0, openfiles-0.5.0, doctestplus-0.5.0, astropy-header-0.1.2
collected 1 item                                                                         

tests/test_consistent.py F                                                         [100%]

======================================== FAILURES ========================================
_______________________________ test_bed_after_one_update ________________________________

    def test_bed_after_one_update():
        """
        """
        delta.update()
        # slice is: delta.eta[:5, 4]
        print(delta.eta[:5, 4])
    
        _exp = np.array([-1., -0.82951754, -0.9981775 , -1., -1.])
>       assert np.all(delta.eta[:5, 4] == pytest.approx(_exp))
E       assert False
E        +  where False = <function all at 0x102778830>(array([-1.       , -0.840265 , -0.9976036, -1.       , -1.       ],\n      dtype=float32) == approx([-1.0 ± 1.0e-06, -0.82951754 ± 8.3e-07, -0.9981775 ± 1.0e-06, -1.0 ± 1.0e-06, -1.0 ± 1.0e-06]))
E        +    where <function all at 0x102778830> = np.all
E        +    and   approx([-1.0 ± 1.0e-06, -0.82951754 ± 8.3e-07, -0.9981775 ± 1.0e-06, -1.0 ± 1.0e-06, -1.0 ± 1.0e-06]) = <function approx at 0x1014f38c0>(array([-1.        , -0.82951754, -0.9981775 , -1.        , -1.        ]))
E        +      where <function approx at 0x1014f38c0> = pytest.approx

tests/test_consistent.py:23: AssertionError
---------------------------------- Captured stdout call ----------------------------------
[-1.        -0.840265  -0.9976036 -1.        -1.       ]
================================ short test summary info =================================
FAILED tests/test_consistent.py::test_bed_after_one_update - assert False
=================================== 1 failed in 0.83s ====================================
```

versus

```
(base) 15:18:33 ericfoot pyDeltaRCM_WMT $ (amoodie/test_consistent) pytest
================================== test session starts ===================================
platform darwin -- Python 3.7.7, pytest-5.4.2, py-1.8.1, pluggy-0.13.1
rootdir: /Users/ericfoot/Dropbox/programs/git_repos/pyDeltaRCM_WMT
plugins: arraydiff-0.3, remotedata-0.3.2, hypothesis-5.11.0, openfiles-0.5.0, doctestplus-0.5.0, astropy-header-0.1.2
collected 102 items                                                                      

tests/test_bmi_delta.py ...                                                        [  2%]
tests/test_command_line_tool.py ...                                                [  5%]
tests/test_consistent.py .                                                         [  6%]
tests/test_deltaRCM_driver.py ..                                                   [  8%]
tests/test_deltaRCM_tools.py ..                                                    [ 10%]
tests/test_init_tools.py ......................................................... [ 66%]
.............                                                                      [ 79%]
tests/test_sed_tools.py .                                                          [ 80%]
tests/test_shared_tools.py ..                                                      [ 82%]
tests/test_water_tools.py .......                                                  [ 89%]
tests/test_yaml_parsing.py ..x..xx....                                             [100%]

============================= 99 passed, 3 xfailed in 7.76s ==============================
```

--- a/pyDeltaRCM/init_tools.py
+++ b/pyDeltaRCM/init_tools.py
@@ -221,7 +221,7 @@ class init_tools(object):
         # self.prefix
         self.prefix = self.out_dir
 
-        if self.out_dir[-1] is not '/':
+        if self.out_dir[-1] != '/':
             self.prefix = self.out_dir + '/'
         if self.site_prefix:
             self.prefix += self.site_prefix + '_'

--- a/tests/test_consistent.py
+++ b/tests/test_consistent.py
@@ -1,0 +1,23 @@
+# unit tests for consistent model outputs
+
+import pytest
+
+import sys
+import os
+import numpy as np
+
+from pyDeltaRCM.deltaRCM_driver import pyDeltaRCM
+
+# need to create a simple case of pydeltarcm object to test these functions
+delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
+
+
+def test_bed_after_one_update():
+    """
+    """
+    delta.update()
+    # slice is: delta.eta[:5, 4]
+    print(delta.eta[:5, 4])
+
+    _exp = np.array([-1., -0.82951754, -0.9981775 , -1., -1.])
+    assert np.all(delta.eta[:5, 4] == pytest.approx(_exp))

--- a/tests/test_consistent.py
+++ b/tests/test_consistent.py
@@ -9,15 +9,15 @@ import numpy as np
 from pyDeltaRCM.deltaRCM_driver import pyDeltaRCM
 
 # need to create a simple case of pydeltarcm object to test these functions
-delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
 
 
 def test_bed_after_one_update():
     """
     """
+    delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
     delta.update()
     # slice is: delta.eta[:5, 4]
-    print(delta.eta[:5, 4])
+    # print(delta.eta[:5, 4])
 
-    _exp = np.array([-1., -0.82951754, -0.9981775 , -1., -1.])
+    _exp = np.array([-1., -0.840265, -0.9976036, -1., -1.])
     assert np.all(delta.eta[:5, 4] == pytest.approx(_exp))

--- a/tests/test_deltaRCM_driver.py
+++ b/tests/test_deltaRCM_driver.py
@@ -9,18 +9,19 @@ import numpy as np
 from pyDeltaRCM.deltaRCM_driver import pyDeltaRCM
 
 # need to create a simple case of pydeltarcm object to test these functions
-delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
 
 
 def test_init():
     """
     test the deltaRCM_driver init (happened when delta.initialize was run)
     """
+    delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
     assert delta._time == 0.
     assert delta._is_finalized == False
 
 
 def test_update():
+    delta = pyDeltaRCM(input_file=os.path.join(os.getcwd(), 'tests', 'test.yaml'))
     delta.update()
     assert delta._time == 1.0
     assert delta._is_finalized == False

--- a/tests/test_yaml_parsing.py
+++ b/tests/test_yaml_parsing.py
@@ -113,6 +113,7 @@ def test_random_seed_settings_value(tmp_path):
     assert delta.seed == 9999
     _postval_same = np.random.uniform()
     assert _preval_same == _postval_same
+    assert delta.seed == 9999
 
 
 def test_random_seed_settings_noaction_default(tmp_path):

